test/test_allreduce_torchrun.py

```diff
@@ -0,0 +1,29 @@
+import os
+import subprocess
+
+def test_local_launch_allreduce():
+    # This test launches a allreduce using torchrun launcher
+    cmd = "torchrun --nproc_per_node=2  --master_addr=127.0.0.1 --master_port=2020 allreduce_torchrun.py "
+
+    proc0 = subprocess.Popen(cmd, shell=True)
```

Comment:
Here and in the test below, can this just be named `proc`?

Reply:
sure..